client/src/components/QandA/SearchQuestion.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const SearchQuestion = (props) => {
+  return (
+    <div className='form-div'>
+      <form onSubmit={props.submit}>
+        <input type = 'text'
+          placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'
+          onChange={props.cQuestion}
+          value={props.questionSearch}
```

Comment:
I believe this should be value={props.qSearch}

Reply:
You are right! Thank!

Reply:
I have updated. We can merge this pull request for now as it is not impacting operations